README.md

```diff
@@ -82,7 +82,11 @@ i18next
   bindI18n: '',
 
   // which events on resourceSource should clear the cache, can be set to false or string of events separated by " "
-  bindI18nStore: ''
+  bindI18nStore: '',
+
+  // Will be run when parser throws an error. Return a string to use in place of the translation.
```

Comment:
I suggest a slight re-phrase: "Return a string to use in place of the translation" => "Can return any string, which can be used as a fallback, in case of broken translation.